tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/LRATypeTckResource.java

```diff
@@ -125,4 +127,9 @@ public Response notSupportedEndLRA(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lra
     public Response neverEndLRA(@HeaderParam(LRA_HTTP_CONTEXT_HEADER) URI lraId) {
         return Response.ok(lraId).build();
     }
-}
\ No newline at end of file
+
+    @AfterLRA
+    public void afterLRA(URI lraId, LRAStatus status) {
+        // no-op, required by the specification
```

Comment:
Since this change should have been included with PR 265 (https://github.com/eclipse/microprofile-lra/pull/265/files#diff-06aca14982a5ee1e4547e9ec14b41bbbR400) maybe you could reference the spec to say why it is required so that we have traceability?

Reply:
ok, I will link the PR in the comment.